sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,7 +1,29 @@
 # Release History
 
-## 7.0.0-beta.8 (Unreleased)
+## 7.0.0 (2020-11-30)
 
+### New Features
+- Exposing enum 'ServiceBusFailureReason' in 'ServiceBusException' which contains set of well-known reasons for an
```

Comment:
"a set of". And suggest remove "that was the cause of an exception" in the following line.